app/libs/classes/viva_application.py

```diff
@@ -88,23 +88,40 @@ def _create_new_application(self):
             'RAWDATATYPE': self._raw_data_type,
         }
 
-        housing = ZeepHousing(
-            application_answer_collection=self._answer_collection)
-        client = housing.get_client(self._personal_number)
-
-        if not client:
-            raise ValueError(f'Client can not be {client}. Verify your tags!')
-
+        household_info = self._get_zeep_household_info()
         attachment_list = self._get_zeep_attachment_list()
         new_application = self._get_zeep_application_dict()
 
-        return {**initial_new_application, **client, **new_application, **attachment_list}
+        return {**initial_new_application, **household_info, **new_application, **attachment_list}
 
     def _save_attachments(self):
         for attachment in self._attachments:
             self._viva_attachments.save(attachment=attachment)
         return True
 
+    def _get_zeep_household_info(self):
+        client_zeep_person_info = ZeepPersonInfo(
+            application_answer_collection=self._answer_collection)
+        client_info = client_zeep_person_info.create()
+
+        if not client_info:
+            raise ValueError(
+                f'Client can not be {client_info}. Verify your tags!')
```

Comment:
Should we really log/stringify `client_info`? Seems like it either always is `None` or we might unintentionally leak info. Perhaps change the message to something like `"Invalid client_info. Verify your tags!"`?